test/dlc_tests/container_tests/bin/pytorch_tests/testPyTorchMpiwithInductor

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+HOME_DIR=/test
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+set -e
+${BIN_DIR}/pytorch_tests/setupPyTorchBackendTest
+cd ${HOME_DIR}
+source dlbs/scripts/environment.sh
+
+set +e
+nvidia-smi
+NVIDIA_SMI_RETURN_VAL=`echo $?`
+set -e
+
+if [ ${NVIDIA_SMI_RETURN_VAL} -eq 0 ]; then
+  echo "Running Gloo test on gpu"
+  NUM_GPU=$(nvidia-smi -L | wc -l)
+  export OMP_NUM_THREADS=${NUM_GPU}
+else
+  echo "Running Gloo test on cpu"
+fi
+
+TRAINING_LOG=${LOG_DIR}/pytorch_train_mpi.log
+
+echo "Training resnet18 using PyTorch Mpi... This may take a few minutes. You can follow progress on the log file : $TRAINING_LOG"
+set +e
+
+sed -i 's/model.train()/& \n    model = torch.compile(model, backend='inductor', mode='default')/g' dlbs/python/pytorch_benchmarks/benchmarks.py
```

Comment:
Add a check to verify if inductor was actually invoked. This substitution will soft fail to eager mode.